client/logic/game/units/unit.gd

```diff
@@ -95,8 +98,8 @@ func get_unit_stats() -> UnitStats:
 func refill() -> void:
 	var sound: GlobalSound = Sound as GlobalSound
 	sound.play("Refill")
-	_unit_stats.fuel = properties.fuel
-	_unit_stats.ammo = properties.ammo
+	_unit_stats.fuel = _types.units[id]["fuel"]
+	_unit_stats.ammo = _types.units[id]["ammo"]
```

Comment:
I will prefer to have the keys as a property, and not a key, the key are easy to fuckup.

Reply:
I agree that properties are a much better choice, but considering that we are in the initial stages of implementing the types, I would suggest creating the property abstraction as a separate step. The type structure is still subject to change.

Reply:
can you create a ticket?